src/styles/forms.scss

```diff
@@ -0,0 +1,76 @@
+.form-button{
+  width: 100px;
+}
+
+#daysUntilCitationExpiresFF{
+  width:50px !important;
+}
+
+.judge-component mat-chip,
```

Comment:
Everything in here that's specific to `judge-component` should probably be in a separate, `judge-component.scss` file...